deploy/template.yaml

```diff
@@ -86,8 +86,8 @@ Mappings:
       MESSAGERETENTIONPERIODDAYS: 345600 # Default: 4 days
       TXMAMESSAGERETENTIONPERIODDAYS: 604800 # Default: 7 days
       GOVUKNOTIFYTEMPLATEID: "bea2a584-4dca-4970-a90d-93977e752fdf"
-      CLIENTSESSIONTOKENTTL: 950400 # Default 11 days
-      RESOURCESTTL: 1036800 # Default 12 days
+      YOTISESSIONTTLDAYS: 10 # Default 10 days
+      RESOURCESTTL: 1209600 # Default 14 days
       CLIENTS: '[{"jwksEndpoint":"https://f2f-ipv-stub-ipvstub.review-o.dev.account.gov.uk/.well-known/jwks.json","clientId":"5C584572","redirectUri":"https://f2f-ipv-stub-ipvstub.review-o.dev.account.gov.uk/redirect"}]'
       AUTHSESSIONTTL: "950400" # 11 days in seconds
```

Comment:
as AUTHSESSIONTTL determines dynamo TTL, it is possible that the original session record gets deleted before 10days if user doesn't finish the whole journey in one go. I think it would be better to reset dynamo TTL once yoti session creation is successful.

Reply:
Done